sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/module-info.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.resourcemanager.compute {
+    requires transitive com.azure.resourcemanager.resources;
+    requires transitive com.azure.resourcemanager.msi;
+    requires transitive com.azure.resourcemanager.network;
+    requires transitive com.azure.resourcemanager.storage;
```

Comment:
Do you need network and storage also to be transitive?

Reply:
In fluent interfaces, we provide convenient configs when creating virtual machine, like [withExistingPrimaryNetwork(Network network)](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java#L475), [withExistingStorageAccount(StorageAccount storageAccount)](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java#L1376). The `Network` comes from module of `com.azure.resourcemanager.network`, and the `StorageAccount` is from `com.azure.resourcemanager.storage`. 

Thus it requires transitive keyword. Otherwise, the end users have to add additional requires statement explicitly when calling `withExistingPrimaryNetwork`/`withExistingStorageAccount`.